### DISCUSSION TITLE
Duvida de instalação

### DISCUSSION THREAD

**Discussion by wiltonwilldepaulo** [swoole/swoole-src#5241] [General]
Boa noite a todos. Peço desculpas pela minha falta de conhecimento, mas gostaria de pedir ajuda com um guia passo a passo para instalar o Swoole no Ubuntu 22.04 Server utilizando o PHP 8.3. Segui as instruções da documentação, mas enfrentei dificuldades. Se alguém tiver um manual disponível, ficarei muito grato por compartilhá-lo. Além disso, também estou buscando orientações para a instalação do Slim Framework 4. Agradeço desde já pela ajuda.

**Comment by danielhe4rt** [comment]
@leocavalcante talvez possa te ajudar nisso ai :p

**Comment by leocavalcante** [comment]
Olá, @wiltonwilldepaulo. Você pode dar mais detalhes de quais foram as dificuldades encontradas durante os processo de instalação?

Como você fez? Foi via `pecl insall swoole`? Foi compliando o código-fonte?
E quais erros apareceram?
